src/Miki/API/Imageboards/Objects/YanderePost.cs

```diff
@@ -1,182 +1,130 @@
-      "file_size": 187671,
-      "file_ext": "jpg",
-      "file_url": "https://files.yande.re/image/da630593cbf6cfe01b868c1ad1632a16/yande.re%20614834%20pantsu%20pantyhose%20shirt_lift%20underboob%20yom.jpg",
-      "is_shown_in_index": true,
-      "preview_url": "https://assets.yande.re/data/preview/da/63/da630593cbf6cfe01b868c1ad1632a16.jpg",
-      "preview_width": 107,
-      "preview_height": 150,
-      "actual_preview_width": 215,
-      "actual_preview_height": 300,
-      "sample_url": "https://files.yande.re/sample/da630593cbf6cfe01b868c1ad1632a16/yande.re%20614834%20sample%20pantsu%20pantyhose%20shirt_lift%20underboob%20yom.jpg",
-      "sample_width": 1073,
-      "sample_height": 1500,
-      "sample_file_size": 217528,
-      "jpeg_url": "https://files.yande.re/image/da630593cbf6cfe01b868c1ad1632a16/yande.re%20614834%20pantsu%20pantyhose%20shirt_lift%20underboob%20yom.jpg",
-      "jpeg_width": 1287,
-      "jpeg_height": 1800,
-      "jpeg_file_size": 0,
-      "rating": "q",
-      "is_rating_locked": false,
-      "has_children": false,
-      "parent_id": null,
-      "status": "active",
-      "is_pending": false,
-      "width": 1287,
-      "height": 1800,
-      "is_held": false,
-      "frames_pending_string": "",
-      "frames_pending": [
-        
-      ],
-      "frames_string": "",
-      "frames": [
-        
-      ],
-      "is_note_locked": false,
-      "last_noted_at": 0,
-      "last_commented_at": 0
-    },
-     */
     public class YanderePost : BooruPost, ILinkable
     {
         public string Url => FileUrl ?? Source;
         public string SourceUrl => Source;
         public string Provider => "Yande.re";
 
-        [DataMember(Name = "created_at")]
+        [JsonProperty("created_at")]
         public int CreatedAt { get; set; }
 
-        [DataMember(Name = "updated_at")]
+        [JsonProperty("updated_at")]
         public int UpdatedAt { get; set; }
 
-        [DataMember(Name = "creator_id")]
+        [JsonProperty("creator_id")]
         public int CreatorId { get; set; }
 
-        [DataMember(Name = "approver_id")]
+        [JsonProperty("approver_id")]
         public object ApproverId { get; set; }
 
-        [DataMember(Name = "author")]
+        [JsonProperty("author")]
         public string Author { get; set; }
 
-        [DataMember(Name = "change")]
+        [JsonProperty("change")]
         public int Change { get; set; }
 
-        [DataMember(Name = "source")]
+        [JsonProperty("source")]
         public string Source { get; set; }
 
-        [DataMember(Name = "md5")]
+        [JsonProperty("md5")]
         public string Md5 { get; set; }
 
-        [DataMember(Name = "file_size")]
+        [JsonProperty("file_size")]
         public int FileSize { get; set; }
 
-        [DataMember(Name = "file_ext")]
+        [JsonProperty("file_ext")]
         public string FileExt { get; set; }
 
-        [DataMember(Name = "file_url")]
+        [JsonProperty("file_url")]
```

Comment:
this property was one of the ones which did not get serialized properly leading to the command not working most of the time